ExampleProject/components/Hello.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from "react"
```

Comment:
this could me 
import React from "react"
now that we have allowSyntheticDefaultImports esModuleInterop enabled.